lib/workers/repository/process/write.ts

```diff
@@ -96,10 +96,10 @@ export async function writeUpdates(
     branch.prBlockedBy = res?.prBlockedBy;
     branch.prNo = res?.prNo;
     branch.result = res?.result;
-    branch.branchFingerprint = res?.commitSha
-      ? branchFingerprint
-      : branchCache.branchFingerprint;
```

Comment:
I thought this line helped to "roll back" if no commit was made.  Now you lost that logic?

Reply:
Ohh yes, I was thinking for a while why I was using ternary operator but couldn't remember. Thanks